package.json

```diff
@@ -50,7 +50,7 @@
     "format": "prettier --list-different \"**/*.{js,md,mdx,json,ts,tsx}\"",
     "format:fix": "prettier --write \"**/*.{js,md,mdx,json,ts,tsx}\"",
     "fresh": "yarn nuke && yarn",
-    "postinstall": "yarn dep:test && preconstruct build && yarn GELjson && yarn build:dynamicblocks",
+    "postinstall": "yarn dep:test && preconstruct dev && yarn GELjson && yarn build:dynamicblocks",
```

Comment:
preconstruct dev had a [bug](https://github.com/preconstruct/preconstruct/issues/484) with TypeScript. cc @PinkTaco97

Reply:
Fixed as part of https://github.com/WestpacGEL/GEL/pull/962